fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -387,6 +387,8 @@ public String logKey() {
         WAIT_SCAN_RECORDS("wait for scan records"),
         /** Wait for a indexed record scan. */
         WAIT_SCAN_INDEX_RECORDS("wait for scan index records"),
+        /** Wait for getting index build states. */
+        WAIT_GET_INDEX_BUILD_STATES("wait for getting index build states"),
```

Comment:
Super pedantic, feel free to ignore, but there is only one build state being waited on, so it should probably be `WAIT_GET_INDEX_BUILD_STATE` and `wait for getting index build state`.